### PR TITLE
583: fix/login by key ui issue

### PR DESCRIPTION
**Pull request by Rus9688**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ straightforward as possible.
 ### Fixed
 - mobile upload on Android
 - device log. new format support [#565]
+- Log in by key uploading UI issue
```
Comment (sergey-lukianov) [inline CHANGELOG.md:38]: Please specify github issue number in []
And add it to PR header as well

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Rus9688) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ straightforward as possible.
 ### Fixed
 - mobile upload on Android
 - device log. new format support [#565]
+- Log in by key uploading UI issue [#583]
```
Comment (sergey-lukianov) [inline CHANGELOG.md:38]: in a fresh release please ....  ie after line 18

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Rus9688) [reply CHANGELOG.md]: done

**Pull request merged**